wp-content/themes/classicpress-susty-child/style.css

```diff
@@ -1749,7 +1749,7 @@ footer#legal {
 
 .home-hero-text h3 {
 	margin-top: 0;
-	font-size: 1.7em;
+	font-size: 1.5em;
```

Comment:
This change was already merged in #33. It appears here again because the `master` branch was not reset correctly for this PR. I am going to change the recommendations above to always use a separate branch for PRs.